src/docs/ganache/corda/nodes.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Ganache | Corda Nodes
+layout: docs.hbs
+---
+# Nodes
+
+<p class="alert alert-info">For Corda technical documentation not related to Ganache head over to [Corda's official documentation](https://docs.corda.net/docs/corda-os/4.4.html).</p>
+
+The Nodes screen lists all the nodes and notaries on the network.
+
+![Corda Nodes](/img/docs/ganache/corda/nodes.png)
+
+Clicking on a node opens the Node Details screen for that node, displaying connection details, [transactions](./transactions), [cordapps](./cordapps), and connected nodes/notaries.
+
+![Corda Node Details](/img/docs/ganache/corda/node-details.png)
+
+
```

Comment:
Should we show the graph view?

Reply:
I don't really like the graph view, which is why I just omitted it entirely. :sweat_smile: